src/System.Management.Automation/engine/ExperimentalFeature/EnableDisableExperimentalFeatureCommand.cs

```diff
@@ -0,0 +1,133 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Management.Automation;
+using System.Management.Automation.Configuration;
+using System.Management.Automation.Internal;
+using System.Management.Automation.Language;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// Base class for Enable/Disable-ExperimentalFeature cmdlet.
+    /// </summary>
+    public class EnableDisableExperimentalFeatureCommandBase : PSCmdlet
+    {
+        /// <summary>
+        /// Gets or sets the feature names.
+        /// </summary>
+        [Parameter(ValueFromPipelineByPropertyName = true, Position = 0, Mandatory = true)]
+        [ArgumentCompleter(typeof(ExperimentalFeatureNameCompleter))]
+        [ValidateNotNullOrEmpty]
+        public string[] Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the scope of persistence of updating the PowerShell configuration json.
+        /// </summary>
+        [Parameter]
+        public ConfigScope Scope { get; set; } = ConfigScope.CurrentUser;
+    }
+
+    /// <summary>
+    /// Implements Enable-ExperimentalFeature cmdlet.
+    /// </summary>
+    [Cmdlet(VerbsLifecycle.Enable, "ExperimentalFeature", SupportsShouldProcess = true, HelpUri = "")]
```

Comment:
Please open a tracking issue to request help links for all cmdlets.

Reply:
generated the uri's and added them